### PR TITLE
Removes references to features on the Analytics tab of the Instructor Dash

### PR DESCRIPTION
**Pull request by lamagnifica**
@dsjen @sarina @shnayder killing AN-4583 with fire
@srpearce this PR removes references to the Analytics tab on the
Instructor Dashboard (and many —but not all— style guide changes
applied to course_answers.rst)

### CODE REVIEW

**Comment by sarina** [comment]
@lamagnifica could you edit the commit message to not `@` mention anyone's github name on the commit? If you merge this to master, we'll all get emails every time anyone commits this commit (ie rebasing, etc). I got two emails, one for this pr and one for your commit.

File: en_us/shared/building_and_running_chapters/running_course/course_answers.rst [hunk 1]
```diff
@@ -5,15 +5,12 @@ Answer Data
...
-* :ref:`Student_Answer_Submission`
-
-* :ref:`Student_Answer_Distribution`
-
-* :ref:`score_histogram`
+.. contents::
+ :local:
+ :depth: 1
 
 Student answer distribution data, including both charts and reports, is also
```
Comment (sarina) [inline en_us/shared/building_and_running_chapters/running_course/course_answers.rst:15]: strike `also`

**Comment by sarina** [comment]
LGTM

File: en_us/release_notes/source/2014/07-21-2014.rst [hunk 2]
```diff
@@ -71,7 +71,7 @@ The following documentation is available:
 
   Recent changes include: 
 
-  * The `Track Student Activity`_ section has been added to the `Student Data`_ chapter.
+  * The Track Student Activity section has been added to the `Student Data`_ chapter.
```
Comment (srpearce) [inline en_us/release_notes/source/2014/07-21-2014.rst:74]: Nit: chapter --> section

**Comment by srpearce** [comment]
:+1:

File: en_us/release_notes/source/2014/07-21-2014.rst (hunk 2 above)
Comment (lamagnifica) [inline en_us/release_notes/source/2014/07-21-2014.rst:74]: note date of file -- I don't think it's worth doing more in that file than removing the dead link

**Comment by shnayder** [comment]
:ship:

**Comment by dsjen** [comment]
LGTM!  :dancer:

**Comment by lamagnifica** [comment]
Needs change log.

**Comment by lamagnifica** [comment]
@mhoeber added change log entries for today's release

**Comment by mhoeber** [comment]
:+1:

**Pull request merged**